scspell/__init__.py

```diff
@@ -572,13 +581,30 @@ def spell_check_file(filename, dicts, ignores, report_only, c_escapes):
             # This is matching the file-id.  Skip over it.
             pos = m_id.end()
             continue
-        result = spell_check_token(MatchDescriptor(data, m),
+        md = MatchDescriptor(data, m)
+        result = spell_check_token(md,
                                    filename, fq_filename, file_id_ref,
-                                   dicts, ignores, report_only)
+                                   dicts, ignores, report_only, list_unique)
+        prev = pos
         (data, pos) = result[0]
         error_found = result[1]
+
         if error_found:
             okay = False
+            if list_unique:
+                err = data[prev:pos]
+                if err in errors:
+                    errors[err].append(md.get_line_num())
+                else:
+                    errors[err] = [md.get_line_num()]
+
+    if list_unique:
+        errors = list(errors.items())
+        errors.sort(key=lambda x: x[1])
+
+        print("Set of mispelled tokens:")
+        for row in errors:
+            print(filename + ":", row[1], ":", row[0].lstrip())
```

Comment:
insead of using `,` , use % notation, like you see in the `print` calls in `report_failed_check`